src/app/stores/migrator.ts

```diff
@@ -372,6 +389,118 @@ export class Migrator {
     return state;
   }
 
+  private replaceUndefinedByNull(value: any): any {
```

Comment:
I'm wondering if we can use a new object with all null properties and then do a deep merge, instead of doing this all maunally.